.github/workflows/workflow.yml

```diff
@@ -196,39 +188,6 @@ jobs:
       - name: Check spelling
         run: tox -e spell_check
 
-  dependencies_checks:
-    continue-on-error: False
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        python_version: [3.7]
-    timeout-minutes: 10
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-python@v3
-        with:
-          python-version: ${{ matrix.python_version }}
-      - if: matrix.os == 'ubuntu-latest'
-        name: Install dependencies (ubuntu-latest)
-        run: |
-          sudo apt-get update --fix-missing
-          sudo apt-get install libmbedtls-dev
-          sudo apt-get autoremove
-          sudo apt-get autoclean
-      - name: Install tox
-        run : |
-          pip install tox
-      - name: Check Pipfile and tox.ini consistency
-        run : |
-          python ./scripts/check_pipfile_and_toxini.py
-      - name: Install current aea
-        run: |
-          pip install -e .[all]
-      - name: Check aea dependenices and imports
-        run: |
-          tox -e dependencies_check
```

Comment:
I think wee can still use this script. Just need to adapt it to pyproject.toml.

Reply:
This script is no compatible with poetry (read setup.py). Need to adapt it to poetry (in another PR) and enable it.